src/javaRosa.js

```diff
@@ -319,6 +320,9 @@ define([
                 }
                 if (!mug.p.helpItext && mug.spec.helpItext.presence !== "notallowed") {
                     mug.p.helpItext = this.createItem();
+                    if (item) {
+                        item.set("", 'markdown');
```

Comment:
This seems odd to me, but I may be missing something. It's clearing the content of the "markdown" form for all languages in the label itext item when creating the help itext item. Is that correct?